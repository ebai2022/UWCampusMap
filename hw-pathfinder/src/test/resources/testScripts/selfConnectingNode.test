# Find a path from a node to itself

CreateGraph g

AddNode g n1

FindPath g n1 n1