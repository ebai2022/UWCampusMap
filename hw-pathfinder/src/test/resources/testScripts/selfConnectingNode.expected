# Find a path from a node to itself

created graph g

added node n1 to g

path from n1 to n1:
total cost: 0.000